packages/next-frontend/src/components/create-campaign/form/pool-step/picker/row/index.tsx

```diff
@@ -0,0 +1,74 @@
+import type { Pool } from "@metrom-xyz/sdk";
+import numeral from "numeral";
+import { Typography } from "@/src/ui/typography";
+import classNames from "@/src/utils/classes";
+import { PoolRemoteLogo } from "@/src/ui/pool-remote-logo";
+import { Skeleton } from "@/src/ui/skeleton";
+
+import styles from "./styles.module.css";
+
+interface PickerRowProps {
+    pool: Pool;
+    style?: any;
+    loading?: boolean;
+    active?: boolean;
+    onClick: (pool: Pool) => void;
+}
+
+export function Row({ style, pool, active, loading, onClick }: PickerRowProps) {
+    function handlePoolOnClick() {
```

Comment:
Not sure if this would benefit from a `useCallback`. Probably yes since pool is an object and I could see its reference changing.

Reply:
Fixed in 73fee9a5f735c6ec90dd504022666e8b724a0de9.